server/db.mjs

```diff
@@ -3,15 +3,81 @@ import pgp from "pg-promise";
 
 const db = initDb();
 
-export const getTasks = async () => await db.any("SELECT * FROM tasks");
+export const getTasks = async () => await db.any("SELECT * FROM park");
 
 export const addTask = async (name) =>
   (
     await db.any("INSERT INTO tasks(name) VALUES($1) RETURNING id, name", [
       name,
     ])
   )[0];
+export const testSearch = async () => {
+  let data = await db.any("SELECT * FROM PARK WHERE PARK_ID IN ('1','2')");
+  console.log(data, "testSearch");
+};
+// testSearch();
+//search usersave parks
+export const getUserFavorites = async (userId) => {
+  const user = await db.any(`SELECT * FROM USERS WHERE USER_ID = '${userId}'`);
```

Comment:
```suggestion
  const user = await db.any(`SELECT * FROM USERS WHERE USER_ID = ‘$1’`, [userId]);
```
Using this format for your database queries might make things easier on you in the future than using template string literals.

Reply:
Thank you Brie for your suggestion!